### PR TITLE
HNP population

### PR DESCRIPTION
**Pull request by Aarushi1104**
(opened the pull request again under a different code since the previous pull request was not letting me commit changes properly)

### CODE REVIEW

File: scripts/world_bank/health_nutrition_population/health_expenditure/expenditure_unittests.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import filecmp
```
Comment (ajaits) [inline scripts/world_bank/health_nutrition_population/health_expenditure/expenditure_unittests.py:1]: Can this be moved to a separate PR for expenditure import?
Let's keep only the population related scripts for this PR.

File: scripts/world_bank/health_nutrition_population/population/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Variables: 
+
+Age: 26
+      
+Gender:2
+
+
+Population
+265 per age per gender
```
Comment (ajaits) [inline scripts/world_bank/health_nutrition_population/population/README.md:25]: 265 countries?

File: scripts/world_bank/health_nutrition_population/population/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+The import process output will be a CSV with the following columns:
+
+Country
+Year
+Gender
+Age
+StatVar
+Population
+
+
```
Comment (ajaits) [inline scripts/world_bank/health_nutrition_population/population/README.md:69]: Pls remove extra blank lines.

File: scripts/world_bank/health_nutrition_population/population/population.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
+import requests
```
Comment (ajaits) [inline scripts/world_bank/health_nutrition_population/population/population.py:1]: Pls add a comment block file header similar to:
https://github.com/datacommonsorg/data/blob/master/scripts/world_bank/wdi/worldbank.py

Please add a block comment describing the script as well.

File: scripts/world_bank/health_nutrition_population/population/README.md (hunk 2 above)
Comment (Aarushi1104) [reply scripts/world_bank/health_nutrition_population/population/README.md:25]: 265 including groups of countries, such as Africa eastern and southern, etc. were there in the original dataset

File: scripts/world_bank/health_nutrition_population/population/population_test.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+mport filecmp
```
Comment (ajaits) [inline scripts/world_bank/health_nutrition_population/population/population_test.py:1]: import.

Pls check if the tests pass.

**Review by ajaits — APPROVED** [review]

**Review by ajaits — APPROVED** [review]
